db/migrate/20161119134300_add_organizer_to_workshops.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrganizerToWorkshops < ActiveRecord::Migration
+  def change
+    add_column :workshops, :organizer, :string
```

Comment:
This really breaks stuff, I don't know why @TimBasel didn't test this. A follow-up PR for this is already on its way, i'll make sure it contains all fixes for this.